pkg/cluster/volumes.go

```diff
@@ -44,16 +44,20 @@ func (c *Cluster) syncVolumes() error {
 		}
 
 		// resize pvc to adjust filesystem size until better K8s support
-		if err = c.syncVolumeClaims(); err != nil {
+		if err = c.syncVolumeClaims(false); err != nil {
 			err = fmt.Errorf("could not sync persistent volume claims: %v", err)
 			return err
 		}
 	} else if c.OpConfig.StorageResizeMode == "pvc" {
-		if err = c.syncVolumeClaims(); err != nil {
+		if err = c.syncVolumeClaims(false); err != nil {
```

Comment:
actually I wonder if we should not call c.syncVolumneClaims everytime without all the ifs here but do it inside the function. But I see that for ebs and off mode we need some extra pre and post treatment here.